cmake/modules/hunter_install_libsodium_config.cmake

```diff
@@ -0,0 +1,20 @@
+include(CMakePackageConfigHelpers)
+include(hunter_test_string_not_empty)
+
+function(hunter_install_libsodium_config)
+
+    hunter_test_string_not_empty("${HUNTER_libsodium_VERSION}")
+    hunter_test_string_not_empty("${HUNTER_PACKAGE_INSTALL_PREFIX}")
+    hunter_test_string_not_empty("${HUNTER_SELF}")
+
+    set(templates "${HUNTER_SELF}/cmake/templates")
+    set(configs "${HUNTER_PACKAGE_INSTALL_PREFIX}/lib/cmake/libsodium")
+
+    write_basic_package_version_file("${configs}/libsodiumConfig.cmake" VERSION "${HUNTER_libsodium_VERSION}" COMPATIBILITY ExactVersion)
+
+    configure_file(
+            "${templates}/libsodiumConfig.cmake.in"
+            "${configs}/libsodiumConfig.cmake"
```

Comment:
so both Release and Debug configs have the same name (?) in this case they will overwrite each other. note that in [this example](https://github.com/forexample/package-example#install-foo) there is common config `FooConfig.cmake` that is the same for both configuration and it loads `FooTargets-release`/`FooTargets-debug`.